classes/class-wc-connect-shipping-label.php

```diff
@@ -226,10 +225,48 @@ public function get_all_items( WC_Order $order ) {
 				}
 
 				$refunded_qty = $order->get_qty_refunded_for_item( $item->get_id() );
+				$remaining_quantity = $item['qty'] - absint( $refunded_qty );
+				/**
+				 * The threshold at which we will start batching items together.
+				 * As an example, if a single order item has a quantity 60, which is more than the default threshold
+				 * value of 20 we will start batching together.
+				 */
+				$threshold = apply_filters( 'wc_connect_shipment_item_quantity_threshold', 20 );
```

Comment:
Do we have an indication at what number of items the issues starts presenting itself? 20 seems like a very low number. If we have data to we could work based on that to determine a threshold closer to the actual limit.

Reply:
actually the number can be pretty high, like in thousands, but this is not based on data, as this is very much dependent on the server's allocated resources and also the resources available for usage by the browser.
On usability side, let's say we make the cut for excess of 60 per item, then we'll be showing 60 rows of for this single item, please see how _quantity_ is converted to a row on the shipping label modal.